infrastructure/code/code_html.go

```diff
@@ -104,6 +105,7 @@ func getCodeDetailsHtml(issue snyk.Issue) string {
 		"RepoCount":          additionalData.RepoDatasetSize,
 		"ExampleCount":       len(additionalData.ExampleCommitFixes),
 		"ExampleCommitFixes": prepareExampleCommitFixes(additionalData.ExampleCommitFixes),
+		"SnykUi":             config.CurrentConfig().SnykUi(),
```

Comment:
suggestion(nitpick): probably would be better for future maintainability to inject a config to avoid potential multi-threading issues.